admin-manual/installation/linux/ubuntu-xenial.rst

```diff
@@ -30,20 +33,28 @@ Install the dependencies
 MySQL
 -----
 
-We strongly recommend using `MySQL <https://www.mysql.com/>`__  5.6 as it's
-much better than its previous major release in terms of speed, scalability and
+We strongly recommend using `MySQL <https://www.mysql.com/>`__  5.6 as it's much
+better than its previous major release in terms of speed, scalability and
```

Comment:
"better than the previous major release" works better than "its previous major release